release/Makefile

```diff
@@ -30,7 +30,7 @@ DRY_RUN=false
 endif
 DATE_YYYYMMDD=$(shell date "+%F")
 WEEKLY?=false
-WEEKLY_BUNDLE_MANIFEST_URL=https://dev-release-assets.eks-anywhere.model-rocket.aws.dev/weekly-releases/bundles/$(DATE_YYYYMMDD)/manifest.yaml
+WEEKLY_RELEASES_URL_PREFIX=https://dev-release-assets.eks-anywhere.model-rocket.aws.dev/weekly-releases
```

Comment:
Didn't think about it before, shouldn't this be on the staging account/bucket?

Reply:
I wanted to keep the staging account as an intermediate between the dev and prod, kind of as an interim location for prod release candidates until they've been qualified for release. This weekly release I see as an "official" dev release so I feel it's more at home in this location.